core/src/main/web/app/mainapp/components/notebook/cell-directive.js

```diff
@@ -217,6 +217,30 @@
         $scope.isCodeCell = function() {
           return $scope.cellmodel.type == 'code';
         };
+
+        $scope.isSection = function() {
```

Comment:
Yeah, you're right,  we are doing that a lot. It's not great and one day we should change it but at least we can stop adding new ones.